Src/AmrCore/AMReX_Interp_3D_C.H

```diff
@@ -381,6 +381,187 @@ face_linear_interp_z (int i, int j, int k, int n, Array4<T> const& fine,
     }
 }
 
+template <typename T>
+AMREX_GPU_DEVICE AMREX_FORCE_INLINE
```

Comment:
I think the reason the CIs fail is that these functions need to be `AMREX_GPU_HOST_DEVICE`.

Reply:
Ah, right. Thanks for catching this!